app/src/main/java/com/summer/practice/tvtracker/Registration/Registration.kt

```diff
@@ -0,0 +1,54 @@
+package com.summer.practice.tvtracker.Registration
+
+import android.content.Intent
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.util.Patterns
+import android.widget.Toast
+import com.summer.practice.tvtracker.MainActivity
+import com.summer.practice.tvtracker.R
+import com.summer.practice.tvtracker.databinding.ActivityLoginBinding
+import com.summer.practice.tvtracker.databinding.ActivityRegistrationBinding
+import com.summer.practice.tvtracker.login.LoginActivity
+
+class Registration : AppCompatActivity() {
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+
+        val binding = ActivityRegistrationBinding.inflate(layoutInflater)
+        val view = binding.root
+        setContentView(view)
+
+        binding.registerButton.setOnClickListener {
+            if (binding.Password.text.isEmpty() or binding.verifyPassword.text.isEmpty()
+                or binding.username.text.isEmpty() or binding.EmailAddress.text.isEmpty()) {
+                val text = "Please fill all fields!"
+                val duration = Toast.LENGTH_SHORT
+                val toast = Toast.makeText(applicationContext, text, duration)
+                toast.show()
```

Comment:
please use string variables instead of hardcoding messages like "Please fill all fields!", u should save this value into strings.xml, also u can resolve the Toast with a single line like "Toast.makeText(applicationContext, "some message or a string variable from strings.xml", Toast.LENGHT_SHORT).show()